usaspending_api/common/exceptions.py

```diff
@@ -18,6 +18,14 @@ class InvalidParameterException(APIException):
     default_code = "invalid_request"
 
 
+class EndpointRemovedException(APIException):
+    """Exception for invalid API request parameters."""
+
+    status_code = status.HTTP_410_GONE
+    default_detail = "Endpoint has been removed"
```

Comment:
Would be nice to provide a more friendly message for any API user who attempts to use these endpoints along with a link to the API documentation.